### PR TITLE
GNSS RLTN receiver and antenna swap

### PR DESCRIPTION
**Pull request by megankortink**
@elidana please double check height in antennas.csv

### CODE REVIEW

File: install/receivers.csv [hunk 1]
```diff
@@ -179,6 +179,7 @@ Trimble Navigation Ltd.,TRIMBLE NETR9,5034K69673,PARW,2018-07-24T23:41:24Z,2019-
 Trimble Navigation Ltd.,TRIMBLE NETR9,5034K69674,LKTA,2011-02-08T01:23:01Z,2018-05-17T02:16:49Z
 Trimble Navigation Ltd.,TRIMBLE NETR9,5034K69674,TGTK,2018-06-26T23:56:38Z,2020-01-29T21:48:37Z
 Trimble Navigation Ltd.,TRIMBLE NETR9,5034K69675,HOKI,2011-02-08T20:13:01Z,2017-07-17T23:48:20Z
+Trimble Navigation Ltd.,TRIMBLE NETR9,5034K69675,RLTN,2020-11-03T00:27:08Z,9999-01-01T00:00:00Z
```
Comment (elidana) [inline install/receivers.csv:182]: please shift down this line (entry after 5034K69675 was undeployed from WAHU)

**Review by elidana — CHANGES_REQUESTED** [review]
Hi @megankortink , can you please change the order of the receivers file so that entries are ordered by receiver make, model, serial number, and deployment time?
thanks! :)

**Review by elidana — APPROVED** [review]
looks good! well done!!!

**Review by elidana — APPROVED** [review]

**Review by CBurton90 — APPROVED** [review]
Looks good!

**Pull request merged**